content/docs/operating/configuration.md

```diff
@@ -150,6 +154,33 @@ names:
 Where `<host>` is a valid hostname.
 
 
+### File based SD configurations `<file_sd_config>`
+
+File based service discovery provides a more dynamic way to configure static targets
+and serves as an interface to plug in custom service discovery mechanisms.
+
+It reads a set of files containing a list of zero or more `<target_group>`s. Changes to
+all defined files are detected via disk watches and applied immediately. Files may be
+provided in YAML or JSON format. Only changes resulting in well-formated target groups
```

Comment:
Could you provide a JSON example or specification somewhere? My guess is that most people will want to use JSON for this purpose, but won't know what it should look like...